onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -28,8 +28,10 @@ class ONNXQuantizer:
     def __init__(self, model, per_channel, reduce_range, mode, static, weight_qType, input_qType, tensors_range,
                  nodes_to_quantize, nodes_to_exclude, op_types_to_quantize, extra_options={}):
 
-        # run shape inference on the model
-        model = onnx.shape_inference.infer_shapes(model)
+        # run shape inference on the model (enabled by default)
+        self.extra_options = extra_options if extra_options is not None else {}
+        if not ('disableShapeInference' in self.extra_options and self.extra_options['disableShapeInference']):
```

Comment:
could you please this option into the doc and comment in quantize.py

Reply:
Added comment in the quantize.py

Reply:
nit: use capital to keep consistent with other options